src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -535,6 +535,36 @@ ucs_status_t uct_rc_mlx5_ep_fence(uct_ep_h tl_ep, unsigned flags)
     return uct_rc_ep_fence(tl_ep, &ep->tx.wq.fi, 1);
 }
 
+static ucs_status_t
+uct_rc_mlx5_ep_flush_keepalive(uct_rc_mlx5_ep_t *ep, unsigned flags,
+                               uct_completion_t *comp)
+{
+    uct_rc_mlx5_iface_common_t *iface = ucs_derived_of(ep->super.super.super.iface,
+                                                       uct_rc_mlx5_iface_common_t);
+
+    UCT_CHECK_PARAM(comp == NULL, "Unsupported completion on keepalive flush");
+
+    /* if something is in processing then do nothing */
```

Comment:
/* if some operations are in progress, no need to send an additional messages */